server/data/migrations/test/20190214184359_questions_refreshrs.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function(knex, Promise) {
+	return knex.schema.createTable('questions_refreshrs', (tbl) => {
+        tbl.increments()
+		tbl
+            .integer('refreshr_id')
+            .unsigned()
+			.references('id')
+			.inTable('refreshrs');
+		tbl
+            .integer('question_id')
```

Comment:
Same deal here w/ the formatting. I would either just either make the changes and push, it'll just overwrite this, or just make note of the formatting and change it later since it's not that big of deal.